### ISSUE TITLE
Forseti 2.0 CLI help shows the wrong binary name ('cli.pyc')

### ISSUE THREAD

**Issue by MartenvanWezel** [GoogleCloudPlatform/forseti-security#1217]
```
$ forseti-client-XXXX-vm@ubuntu> forseti
usage: cli.pyc [-h] [--endpoint ENDPOINT] [--use_model USE_MODEL][--out-format {text,json}]
 {explainer,playground,inventory,config,model,scanner,notifier} 
```

Suggest to override cli.pyc to 'forseti'.

**Comment by blueandgold** [comment]
Should be a quick fix.

**Comment by joecheuk** [comment]
Resolved #1246
